product_custom_attribute/views/product.xml

```diff
@@ -1,23 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data>
 
-        <record model="ir.ui.view" id="product_template_form_view">
+        <record id="product_template_form_view" model="ir.ui.view">
             <field name="name">attributes.product.normal.form</field>
             <field name="model">product.template</field>
             <field name="inherit_id" ref="product.product_template_form_view" />
             <field name="arch" type="xml">
                 <field name="type" position="after">
-                    <field name="attribute_set_id" context="{'default_model_id': %(product.model_product_template)d}"/>
+                    <field name="attribute_set_id"
```

Comment:
can you move this field into the tab of attribute. It's will be improve the UI

Reply:
Thinking of it, don't you think it would be even better to put this attribute_set field under the Product's name, to follow the position of the `parent_id`field in Contacts form view :
![image](https://user-images.githubusercontent.com/31664455/77775088-d43e2000-702a-11ea-9abf-cbd7a1084575.png)